### ISSUE TITLE
python2 parser bug: lambda (k,v)

### ISSUE THREAD

**Issue by ievans** [semgrep/semgrep#994]
reproduction: https://semgrep.live/KxkY

this line is not valid python3, but it is valid python2:
```return dict(map(lambda (k,v): (k,self.format(v)), val.iteritems()))```

**Comment by JBthePenguin** [comment]
I'm not sure if that's what you really want, but for python3 this should work.
```python
return dict(map(lambda args: (args[0],self.format(args[1])), val.items()))
```

**Comment by nbrahms** [comment]
Also fixes returntocorp/semgrep#1164.
